common/scripts/restart.sh

```diff
@@ -33,10 +33,10 @@ __start_secrets() {
 
   if [ "$secrets_ip" == "$ADMIRAL_IP" ]; then
     __process_msg "Checking secrets container"
-    local secrets_container=$(sudo docker ps -a -q -f "name=secrets" | awk '{print $1}')
-    if [ "$secrets_container" != "" ]; then
+    local secrets_container=$(sudo docker ps -a --format "{{.Names}}" | grep -w "secrets") || true
+    if [ ! -z "$secrets_container" ]; then
       __process_msg "Found a stopped secrets container, starting it"
-      sudo docker start $secrets_container
+      sudo docker start secrets
```

Comment:
can we make `secrets` as a variable then use it in the grep and docker start command. ideally the secrets image name, container name should be stored up top but if that requires lot of changes, the least we should do it make the functions clean